webcompat/api/endpoints.py

```diff
@@ -135,33 +120,27 @@ def get_issue_category(issue_category):
         # add "status-" before the filter param to match the naming scheme
         # of the repo labels.
         params['labels'] = 'status-' + issue_category
-        if g.user:
-            issues = github.raw_request('GET', issues_path, params=params)
-        else:
-            issues = proxy_request('get', params=params)
+        return api_request('get', issues_path, params=params)
     elif issue_category == 'closed':
         params['state'] = 'closed'
-        if g.user:
-            issues = github.raw_request('GET', issues_path, params=params)
-        else:
-            issues = proxy_request('get', params=params)
+        return api_request('get', issues_path, params=params)
     # Note that 'new' here is primarily used on the homepage.
     # For paginated results on the /issues page, see /issues/search/new.
     elif issue_category == 'new':
-        if g.user:
-            issues = github.raw_request('GET', issues_path, params=params)
-        else:
-            issues = proxy_request('get', params=params)
-        # Do not send random JSON to filter_new
-        if issues.status_code == 200:
-            return (filter_new(json.loads(issues.content)),
-                    issues.status_code, get_headers(issues))
-        else:
-            return ({}, issues.status_code, get_headers(issues))
+        issues = api_request('get', issues_path, params=params)
+        # api_request returns a tuple of format:
+        #       (content, status_code, response_headers)
+        # So we make a dict here for improved readability
+        new_issues = {
+            'content': filter_new(json.loads(issues[0])),
+            'status_code': issues[1],
+            'response_headers': issues[2]
+        }
```

Comment:
ah! I didn't think it would be here, but more in api_request. OK let's move forward and let's see if we can fix that later. ^_^